docs: Fixed introduction date for ceil/ceiling function

peter-b opened the pull request:
The `ceil`/`ceiling` function wasn't available in LiveCode 6.7.  It
was in fact introduced in commit d919382b, which was included in
LiveCode 7.1.

livecodepanos commented:
@peter-b could you please apply the same changes to the `floor` function as well? Both of them were introduced in LC 7.1

peter-b committed:
docs: Fixed introduction date for floor/ceiling functions

The `floor` and `ceil`/`ceiling` functions weren't available in
LiveCode 6.7.  They were in fact introduced in commit d919382b,
which was included in LiveCode 7.1.

peter-b commented:
@livecodepanos Thanks for spotting that -- I've updated the patch.

livecodepanos commented:
thanks @peter-b 

@livecode-vulcan review ok 9a7af85

livecode-vulcan commented:
💙 review by @livecodepanos ok 9a7af85a726141cc1c819721769e1c9cca7dcb7e

livecode-vulcan commented:
😎 test success 9a7af85a726141cc1c819721769e1c9cca7dcb7e

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/629)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/634)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/629)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/633)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/638)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/630)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/630)
